viewtree.html

```diff
@@ -57,7 +57,11 @@
         <div class="container" style="width:100%">
         <section id="upload3" class="upload" >
             <p>Optional Config File:</p>
-            <input style="float:left"   type="file" value="catib.config" name="configFile" id="configFile"/>
+            <select id="config_file_selector">
+                <option>Choose File</option>
+                <option>Upload your config file</option>
+            </select>
+            <input style="float:left; display:none" type="file" name="configFile" id="configFile"/>
```

Comment:
When defining IDs sometimes camel case is used, while other times snake case is used. Please be consistent.

Reply:
I will consistently define ids in camel case (as is with the original code)